README.rst

```diff
@@ -37,21 +39,51 @@ This code runs on Python 3.5 or newer.
 Docker
 ------
 
-Alternatively, you can build and run the xblock-sdk in Docker.
+Alternatively, you can build and run the xblock-sdk in Docker (we are using docker-compose which
+can be installed as explained at https://docs.docker.com/compose/install/)
 
 After cloning this repository locally, go into the repository directory and build the Docker image::
 
-    $ docker build -t xblock-sdk .
+    $ make docker_build
+
+or manually run
+
+    $ docker-compose build
 
 You can then run the locally-built version using the following command::
 
-    $ docker run -d -p 8000:8000 --name xblock-sdk xblock-sdk
+    $ make dev.up
+
+or manually run
+
+    $ docker-compose up -d
+
+and shut-down the container by::
+
+    $ make dev.down
```

Comment:
Since this deletes state, in general use `stop` is what is preferred so let's change to that.

Reply:
Done, with an addition about `down` in addition to `stop`.